target/linux/sunxi/image/cortex-a7.mk

```diff
@@ -31,6 +31,16 @@ define Device/friendlyarm_nanopi-m1-plus
 endef
 TARGET_DEVICES += friendlyarm_nanopi-m1-plus
 
+define Device/sun8i-h3-nanopi-r1
```

Comment:
Please have a look at the other devices here. The name should be the compatible with "," replaced by "_", i.e. friendlyarm_nanopi-r1